src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx

```diff
@@ -0,0 +1,257 @@
+import "./BloodPressureVisualization.css";
+import { useEffect, useState } from "react";
+import { debounce } from "lodash";
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import plus from './plus.svg';
+import React from "react";
+import { Button, DateRangeNavigator, LoadingIndicator } from "../../presentational";
+import DumbbellChart from "../../presentational/DumbbellChart";
+import { Dumbbell, ClosedInterval, Axis, DataPoint, DumbbellClass } from "../../presentational/DumbbellChart/DumbbellChart";
+import { previewSurveyData } from './BloodPressureVisualization.previewdata'
+import { addDays, format, isEqual, isMonday, nextSunday, parseISO, previousMonday, startOfDay } from "date-fns";
+
+export type DataFaucet = "PreviewSurvey" | "Survey";
+
+export interface BloodPressureVisualizationProps {
+    source : DataFaucet,
```

Comment:
Let's follow convention and just use "previewState" like other components to put in some preview data.  That needs to be independent of what data sources this is configured to use in reality anyways